fractal_server/images/models.py

```diff
@@ -70,6 +70,25 @@ def validate_attributes(
         return v
 
 
+class SingleImageUpdate(SingleImageBase):
```

Comment:
do not inherit, since it's not really worth it here
(and we'd have to exclude origin)